psrth/psrth.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Parth Sharma
+institution: The Shri Ram School, Moulsari
+profile_pic: psrth.png
+quote: No matter how you get there or where you end up, human beings have this miraculous gift to make that place home.
```

Comment:
Please shorten your quote so that it meets the character length requirement (100 characters max, current character count: 112).